backend/dataall/modules/dataset_sharing/api/input_types.py

```diff
@@ -10,10 +9,10 @@
         gql.Argument(name='principalId', type=gql.NonNullableType(gql.String)),
         gql.Argument(name='principalType', type=gql.NonNullableType(gql.String)),
         gql.Argument(name='requestPurpose', type=gql.String),
+        gql.Argument(name='attachMissingPolicies', type=gql.Boolean)
```

Comment:
Minor: this field should be NonNullable right?

Reply:
changed